app/views/admin/users.py

```diff
@@ -3,7 +3,7 @@
 
 from . import bp
 from app import create_logger
-from app.forms.user import UserForm
+from app.forms.user import UserForm, CreateUserForm
```

Comment:
I don't think we need to rename to `EditUserForm` since create form is the only specific use case.